src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/run.py

```diff
@@ -67,25 +71,57 @@ def acr_run(cmd,  # pylint: disable=too-many-locals
         source_location = check_remote_source_code(source_location)
         logger.warning("Sending context to registry: %s...", registry_name)
 
+    yaml_template = None
```

Comment:
`az acr run -r MyRegistry -f acb.yaml /dev/null` return error `Some of the properties of 'runRequest' are invalid. InnerErrors: FileTaskRunRequest:Missing required property 'SourceLocation'.`

If the user passes `/dev/null`, EncodedTaskRunRequest should be sent.

Reply:
Is it a workaround? What about enabling none for source location of FileTaskRunRequest API?